var/spack/repos/builtin/packages/r-impute/package.py

```diff
@@ -10,8 +10,8 @@ class RImpute(RPackage):
     """Imputation for microarray data (currently KNN only)."""
 
     homepage = "https://www.bioconductor.org/packages/impute/"
-    git      = "https://git.bioconductor.org/packages/impute.git"
+    url      = "https://bioconductor.org/packages/release/bioc/src/contrib/impute_1.56.0.tar.gz"
 
-    version('1.50.1', commit='31d1cc141797afdc83743e1d95aab8a90ee19b71')
+    version('1.56.0', sha256='6328aaddbf6a7c9af53afb3d6155a927c7a3a6c7c7486e8f493896df82bce023')
```

Comment:
Could you please add this as a new version instead of replacing ?

Reply:
added as new version, points to bioconductor 3.8 release